src/openforms/js/components/admin/form_design/registrations/objectsapi/utils.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const getFieldErrors = (name, index, errors, field) => {
+  const errorMessages = [];
+
+  for (const [errorName, errorReason] of errors) {
+    if (errorName.startsWith(name)) {
+      const errorNameBits = errorName.split('.');
+      if (errorNameBits[2] === String(index) && errorNameBits[errorNameBits.length - 1] === field) {
```

Comment:
```suggestion
      if (errorNameBits[2] === String(index) && errorNameBits.at(-1) === field) {
```
Can we safely make use of recent features? (i.e. is there any polyfills in the build)

Reply:
we use `browserslist` which informs babel about the required transforms, so I think it's okay and will be transpiled away

browserslist config is:

```
  "browserslist": [
    ">0.5%",
    "not dead",
    "not op_mini all"
  ],
```
in `package.json`